ls1intum/Artemis
# `Exam mode`: Fix an issue during the reset of quiz exercises

### Checklist
#### General


#### Server

### Motivation and Context
Resetting an exam does also set the release date of quiz exercises.
This causes issues when importing an exam as exercises that should be imported in an exam must not have any dates set.

### Description
Do not set the release date of quiz exercises in exam.

### Steps for Testing
1. Create an exam with a modelling exercise
2. Reset this exam
3. Import this exam again -> It should work

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|
